packages/fondue/tailwind.config.ts

```diff
@@ -124,6 +124,7 @@ export default {
                 width: 'width',
             },
             screens: {
+                '2xs': '390px',
```

Comment:
Which occasion do you need such a small breakpoint?

Reply:
This is example in the search dialog: to 390 is one pic, over 390 to 480 is 2 pics and more then 480 is 3 pics.
<img width="512" alt="Screenshot 2024-05-06 at 14 33 10" src="https://github.com/Frontify/fondue/assets/517677/08701e21-17dd-4812-b245-e500239566cb">
<img width="559" alt="Screenshot 2024-05-06 at 14 33 22" src="https://github.com/Frontify/fondue/assets/517677/226e5448-7a9b-4490-b4a6-93eead5e03d9">

I was looking at this to see how many are used at this resolution. 

<img width="770" alt="Screenshot 2024-05-06 at 14 31 34" src="https://github.com/Frontify/fondue/assets/517677/10aa44b5-cb08-4d12-97da-d52634f76449">

Reply:
@SamuelAlev Would you go a bit bigger then 390?

Reply:
the card is a bit big to have it till 480px. or to have till 1280px only two cards.
I wanted to do `tw-grid-rows-1 2xs:tw-grid-rows-2 xs:tw-grid-rows-3`

Reply:
Viewport of popular devices Like here: you have them under 390px and above 412 to 480. 
https://www.peeayecreative.com/docs/divi-responsive-helper/popular-device-viewport-sizes/

Reply:
We will move the `xs` to `390px` so you don't have to create `2xs`

Reply:
@SamuelAlev I pushed the change. Is that what you are refering?